site_scons/components/__init__.py

```diff
@@ -1,4 +1,6 @@
 # Copyright 2016-2024 Intel Corporation
+# Copyright 2025 Google LLC
```

Comment:
Looks like you also need this
https://github.com/daos-stack/daos/commit/8905322fe97761cb936450cef5f74f0afbf55c3a

Reply:
Thanks, was looking at the other ignore list and not finding it.